flowpipe/plug.py

```diff
@@ -130,7 +193,7 @@ def __init__(self, name, node):
             raise ValueError(
                 'Names for plugs can not contain dots "." as these are '
                 'reserved to identify sub plugs.')
-        super(OutputPlug, self).__init__(name, node, (InputPlug, SubInputPlug))
+        super(OutputPlug, self).__init__(name, node)
```

Comment:
Why are the accepted_plugs removed?

Reply:
Because the `accepted_plugs` and everything related to that now resides in the `_InPlug` / `_OutPlug` classes.